extensions/amp-analytics/0.1/vendors.js

```diff
@@ -59,6 +64,17 @@ export const ANALYTICS_CONFIG = {
           'st=${socialActionTarget}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'parsely': {
+    'host': 'srv.pixel.parsely.com',
+    'method': 'GET',
+    'requests': {
+      'baseHit': '/plogger/?rand={{timestamp}}&idsite={{apikey}}&url={{ampdocUrl}}&urlref={{documentReferrer}}&screen={{screenWidth}}x{{screenHeight}}%7C{{screenAvailWidth}}x{{screenAvailHeight}}%7C{{colorDepth}}&title={{title}}&date={{timestamp}}&ampid={{clientId}}'
+      'pageview': '{{baseHit}}&action=pageview',
```

Comment:
The first comma is, indeed, crucial. We do, however, prefer the trailing comma in literals. Need to check the linter to actually enforce that.